src/main/java/org/sagebionetworks/web/client/view/users/RegisterAccountViewImpl.java

```diff
@@ -55,8 +55,8 @@
 	Presenter presenter;
 	private Header headerWidget;
 	public static final String ROOT_PORTAL_URL = Window.Location.getProtocol() + "//" + Window.Location.getHost() + "/";
-	public static final String GOOGLE_OAUTH_CALLBACK_URL = RegisterAccountViewImpl.ROOT_PORTAL_URL + "Portal/oauth2callback?oauth2provider=GOOGLE_OAUTH_2_0";
-	public static final String GOOGLE_OAUTH_WITH_STATE_CALLBACK_URL = GOOGLE_OAUTH_CALLBACK_URL + "&state=";
+	public static final String OAUTH_CALLBACK_URL = RegisterAccountViewImpl.ROOT_PORTAL_URL + "Portal/oauth2callback?oauth2provider=";
+	public static final String GOOGLE_OAUTH_WITH_STATE_CALLBACK_URL = OAUTH_CALLBACK_URL + "GOOGLE_OAUTH_2_0&state=";

```

Comment:
During registration, however, we currently only support Google-based user creation.  If we will support others, the registration widget (in SWC) will need to be updated, and this would also need to be changed.  We may want to reimplement the registration widget as a react component at that time.